mem0/configs/embeddings/base.py

```diff
@@ -0,0 +1,25 @@
+from abc import ABC
+from typing import Optional
+
+class BaseEmbedderConfig(ABC):
+    """
+    Config for embedding models
+    """
+    def  __init__(
+        self,
+        model: Optional[str] = None,
+        dims: Optional[int] = None,
+        model_kwargs: Optional[dict] = None
+    ):
+        """
```

Comment:
Assuming model as deployment_name in config args for azure.
Let me know if I should explicitly declare "deployment_name" instead for Azure OpenAI models.